pipeline/src/lib.rs

```diff
@@ -41,10 +48,26 @@ pub async fn session_from_config<F: ConclusionFeed + 'static>(
         .with_information_schema(true);
 
     let mut ctx = SessionContext::new_with_config(session_config);
-    ctx.register_table(
-        "conclusion_feed",
-        Arc::new(conclusion::FeedTable::new(config.conclusion_feed)),
-    )?;
+    match config.conclusion_feed {
+        ConclusionFeedSource::Direct(conclusion_feed) => {
+            ctx.register_table(
+                "conclusion_feed",
+                Arc::new(conclusion::FeedTable::new(conclusion_feed)),
+            )?;
+        }
+        #[cfg(test)]
+        ConclusionFeedSource::InMemory(table) => {
+            debug_assert_eq!(
+                schemas::conclusion_feed(),
+                datafusion::catalog::TableProvider::schema(&table)
+            );
+            ctx.register_table("conclusion_feed", Arc::new(table))?;
+        }
+    };
+    // Register the _internal schema
+    ctx.catalog("ceramic")
+        .unwrap()
```

Comment:
maybe `expect` since it seems like the ceramic catalog must exist? my preference is generally unwrap in tests and expect for "cannot fail/asserts" but again, fine to just close as is since we add it to the session above.